engine-api/src/schema/block.rs

```diff
@@ -0,0 +1,93 @@
+use {
+    moved::{
+        primitives::{Address, Bytes, B2048, B256},
+        types::state::{BlockResponse, TransactionResponse},
+    },
+    serde::{Deserialize, Serialize},
+};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GetBlockResponse {
```

Comment:
According to the docs `baseFeePerGas` and `mixHash` parameters are missing.
https://www.quicknode.com/docs/optimism/eth_getBlockByHash

When I made a call to this API while running the integration test, I see the following params are missing:
`baseFeePerGas`, `blobGasUsed`, `excessBlobGas`, `mixHash`, `parentBeaconBlockRoot`, `withdrawals`, `withdrawalsRoot`

Reply:
Is there a type from the alloy library that can be used here instead of defining it ourselves?

Reply:
I think [alloy-rpc-types-eth](https://github.com/alloy-rs/alloy/tree/main/crates/rpc-types-eth) crate has a lot of the receipt structs.

Reply:
I think that receipt struct is specific to native Ethereum but needs to be extended with Optimism specific fields.